types/tlds/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for tlds 1.199
+// Project: https://github.com/stephenmathieson/node-tlds
+// Definitions by: Ajay Gopal Shrestha <https://github.com/ajshres>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+/*
+Declaring const tlds and exporting it
+*/
+export const tlds: string[];
```

Comment:
Looking at the source of node-tlds, it's a commonjs export, which means it needs to be exported with `export =` like so

```ts
declare const tlds: string[];
export = tlds
```

and imported with `import tlds = require('tlds')`